site/content/user-guide/jmeter_load_tests.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "JMeter Load Tests"
+weight = 300
```

Comment:
"Developer Guide" under "User Guide" in the structure already starts with weight=250 (config.toml). The weights of the MDs inside a drawer seems to be isolated.